tests/e2e_testdata.go

```diff
@@ -0,0 +1,514 @@
+package tests
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/adshmh/meter/db"
+	"github.com/gojektech/heimdall/httpclient"
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	influxAPI "github.com/influxdata/influxdb-client-go/v2/api"
+	influxDomain "github.com/influxdata/influxdb-client-go/v2/domain"
+	"github.com/pokt-foundation/portal-api-go/repository"
+	stringUtils "github.com/pokt-foundation/utils-go/strings"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO move this to utils-go
+func startOfDay(day time.Time) time.Time {
+	y, m, d := day.Date()
+	l := day.Location()
+
+	return time.Date(y, m, d, 0, 0, 0, 0, l)
+}
+
+var (
+	// Test vars
+	ctx               = context.Background()
+	today             = startOfDay(time.Now().UTC())
+	endOfDay          = today.AddDate(0, 0, 1)
+	todayFormatted    = today.Format(time.RFC3339)
+	endOfDayFormatted = endOfDay.Format(time.RFC3339)
+	influxOptions     = db.InfluxDBOptions{
+		URL:                 "http://localhost:8086",
+		Token:               "mytoken",
+		Org:                 "myorg",
+		DailyBucket:         "mainnetRelayApp1d",
+		CurrentBucket:       "mainnetRelayApp10m",
+		DailyOriginBucket:   "mainnetOrigin1d",
+		CurrentOriginBucket: "mainnetOrigin60m",
+	}
+	// Test HTTP client
+	testClient = httpclient.NewClient(httpclient.WithHTTPTimeout(5*time.Second), httpclient.WithRetryCount(0))
+	// Test errors
+	errResponseNotOK = errors.New("Response not OK")
+
+	// Selection of relays to populate the test InfluxDB (selected from this slice inside populateInfluxRelays)
+	testRelays = []struct {
```

Comment:
Would it make the code more readable if the data here, i.e. all the test relays, were in a json-formatted file that is read/unmarshalled into this array? This would allow updating the test data by editing the dataset in json, rather than modifying GO code.

Reply:
I don't have a strong opinion about it either way. There are already JSON formatted strings in this Go file (necesary to subtitute variables into the string) so my preference is to have everything in one test data file rather than multiple files of different formats but ultimately I'm not strongly attached. WDYT?

Reply:
moved them to a JSON file